.github/workflows/cd.yml

```diff
@@ -75,32 +72,71 @@ jobs:
           merge-multiple: true
           path: dist/
       - run: ls -lh dist/
+      # TODO: some more checks? `twine` is already run above
+
+  version_tag:
+    outputs:
+      version: ${{ steps.version.outcome == 'success' && steps.version.outputs.full || '0.0.0' }}
+      prerelease: ${{ steps.version.outcome == 'success' && steps.version.outputs.prerelease || '' }}
```

Comment:
that doesn‘t make sense, we need to decide if we want to fall back to `''` or `'0.0.0'`. This does either depending on if there is a tag or if the tag doesn’t match.